src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -74,8 +77,14 @@ export const validationSchema_v2 = Yup.object()
   .shape<SchemaShape>({
     name: Yup.string()
       .required("Name is required!")
-      .min(3, ({ min }) => `Name must be at least ${min} characters`)
-      .max(25, ({ max }) => `Name must be less than ${max} characters`)
+      .min(
+        VENUE_NAME_MIN_CHAR_COUNT,
+        ({ min }) => `Name must be at least ${min} characters`
+      )
+      .max(
+        VENUE_NAME_MAX_CHAR_COUNT,
+        ({ max }) => `Name must be less than ${max} characters`
+      )
```

Comment:
Why didn't we just use `roomTitleSchema` here directly, rather than repeating the code?

Reply:
See changes in https://github.com/sparkletown/sparkle/pull/1341